eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -95,14 +95,11 @@ end {
 }
 
 begin {
-    # https://review.learn.microsoft.com/help/platform/metadata-taxonomies?branch=main#product
+    # https://taxonomy.learn.microsoft.com/TaxonomyServiceAdminPage/#/taxonomy/
```

Comment:
I don't see the list at this url. Is there some other way to see the new list?

Reply:
The only way to see the new list is to link to a specific run, but I don't think we should do that since there is a new run everyday, and I don't know how far back they keep the history